src/features/ValueEditor/ValueEditor.tsx

```diff
@@ -81,7 +84,7 @@ export const ValueEditor = ({
         justify="space-between"
       >
         <Flex gap="sm" align="center">
-          Value #{index}
+          Value #{index + 1}
```

Comment:
At a presentational level we do the addition now

Reply:
No concern about perf